aep/general/0005/aep.md

```diff
@@ -0,0 +1,84 @@
+# Designing an API
+
+This AEP serves as a high-level guide on designing an API that adheres to the
+AEPs.
+
+## Process summary
+
+1. enumerate the user case would like your API to satisfy.
+1. identify resources and add them to your API.
+1. identify standard operations and add them to your API.
+1. identify custom operations and add them to your API.
+
+## Enumerate use cases
+
+The first step in designing an API is understanding precisely what you would
+like your user to be able to accomplish your API.
+
+Enumerate one or more use cases to clarify what precisel
```

Comment:
I think you accidentally part of this sentence.

Reply:
Doh! Thanks and sorry.